tests/include/test/ssl_helpers.h

```diff
@@ -118,6 +118,9 @@ typedef struct mbedtls_test_handshake_test_options {
 #if defined(MBEDTLS_SSL_CACHE_C)
     mbedtls_ssl_cache_context *cache;
 #endif
+#if defined(MBEDTLS_SSL_ALPN)
+    const char *alpn_list[MBEDTLS_SSL_MAX_ALPN_LIST_LEN/MBEDTLS_SSL_MAX_ALPN_NAME_LEN];
```

Comment:
That's a lot, in ssl_client2/server2 it is limited to 10 (ALPN_LIST_SIZE). We should probably do something similar here.